src/cargo/ops/cargo_install.rs

```diff
@@ -317,6 +317,14 @@ impl<'gctx> InstallablePackage<'gctx> {
         self.gctx.shell().status("Installing", &self.pkg)?;
 
         let dst = self.root.join("bin").into_path_unlocked();
```

Comment:
It seems clearer if we consistently use absolute paths everywhere, not just for display purposes. I believe our assumption here is that it’s always an absolute path.